module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -402,7 +403,10 @@ public function addAccountBlocksToFlashMessenger($catalog, $patron)
             && $blocks = $catalog->getAccountBlocks($patron)
         ) {
             foreach ($blocks as $block) {
-                $this->flashMessenger()->addMessage($block, 'error');
+                $this->flashMessenger()->addMessage(
+                    [ 'msg' => $block, 'dataset' => [ 'lightbox-ignore' => true ] ],
```

Comment:
Does it make sense to put a boolean true here, since this is going to get concatenated into a string later? Would it be better to use a string at this point instead of relying on PHP's type conversion?